extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/model/SmugMugImageUploadResponse.java

```diff
@@ -26,26 +26,22 @@
   @JsonProperty("method")
   private String method;
 
-  @JsonProperty("image")
+  @JsonProperty("Image")
   private ImageInfo image;
 
-  @JsonProperty("code")
```

Comment:
If we get rid of these, is there a risk of not being able to deserialize the response?